app-modules/multifactor-authentication/src/Settings/MultifactorSettings.php

```diff
@@ -44,6 +44,6 @@ class MultifactorSettings extends Settings
 
     public static function group(): string
     {
-        return 'multifactor';
+        return 'authentication';
```

Comment:
This cannot and should not be changed. This group is related to the data saved into the database. Not the navigation.